test/geometry/camera/test_projections.py

```diff
@@ -0,0 +1,225 @@
+        self._test_jit_project(device, dtype)
+        self._test_jit_unproject(device, dtype)
+
+
+class TestProjectionOrthographic(BaseTester):
+    def test_smoke(self, device, dtype):
+        points = torch.tensor([1.0, 2.0, 3.0], device=device, dtype=dtype)
+        assert project_points_orthographic(points) is not None
+
+    def _test_cardinality_unproject_batch(self, device, dtype, batch_size):
+        batch_tuple = (batch_size,) if batch_size is not None else ()
+        points = torch.rand(batch_tuple + (3,), device=device, dtype=dtype)
+        assert project_points_orthographic(points).shape == batch_tuple + (2,)
+
+    def _test_cardinality_project_batch(self, device, dtype, batch_size):
+        batch_tuple = (batch_size,) if batch_size is not None else ()
+        points = torch.rand(batch_tuple + (2,), device=device, dtype=dtype)
+        extension = torch.rand(batch_tuple, device=device, dtype=dtype)
+        assert unproject_points_orthographic(points, extension).shape == batch_tuple + (3,)
+
+    @pytest.mark.parametrize("batch_size", [None, 1, 2, 3])
+    def test_cardinality(self, device, dtype, batch_size):
+        self._test_cardinality_project_batch(device, dtype, batch_size)
+        self._test_cardinality_unproject_batch(device, dtype, batch_size)
+
+    def test_project_points_orthographic(self, device, dtype):
+        points = torch.tensor([1.0, 2.0, 3.0], device=device, dtype=dtype)
+        expected = torch.tensor([1.0, 2.0], device=device, dtype=dtype)
+        self.assert_close(project_points_orthographic(points), expected)
+
+    def test_project_points_orthographic_batch(self, device, dtype):
+        points = torch.tensor([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]], device=device, dtype=dtype)
+        expected = torch.tensor([[1.0, 2.0], [4.0, 5.0]], device=device, dtype=dtype)
+        self.assert_close(project_points_orthographic(points), expected)
+
+    def test_unproject_points_orthographic_extension(self, device, dtype):
+        points = torch.tensor([1.0, 2.0], device=device, dtype=dtype)
+        extension = torch.tensor([2.0], device=device, dtype=dtype)
+        expected = torch.tensor([1.0, 2.0, 2.0], device=device, dtype=dtype)
+        self.assert_close(unproject_points_orthographic(points, extension), expected)
+
+    def test_unproject_points_orthographic_batch_extension(self, device, dtype):
+        points = torch.tensor([[1.0, 2.0], [3.0, 4.0]], device=device, dtype=dtype)
+        extension = torch.tensor([2.0, 3.0], device=device, dtype=dtype)
+        expected = torch.tensor([[1.0, 2.0, 2.0], [3.0, 4.0, 3.0]], device=device, dtype=dtype)
+        self.assert_close(unproject_points_orthographic(points, extension), expected)
+
+    def test_dx_proj_x(self, device, dtype):
+        points = torch.tensor([1.0, 2.0, 3.0], device=device, dtype=dtype)
+        expected = torch.tensor([1.0], device=device, dtype=dtype)
+        self.assert_close(dx_project_points_orthographic(points), expected)
+
```

Comment:
add project_unproject
```suggestion

    def test_project_unproject_points_orthographic(self, device, dtype):
        points = torch.tensor([1.0, 2.0, 3.0], device=device, dtype=dtype)
        extension = torch.tensor([3.0], device=device, dtype=dtype)
        self.assert_close(unproject_points_orthographic(project_points_orthographic(points), extension), points)

```